Submodule ModLoader contains modified content
Submodule mod/BeautySelectorAddon contains modified content
diff --git a/ModLoader/mod/BeautySelectorAddon/src/BeautySelectorAddon.ts b/ModLoader/mod/BeautySelectorAddon/src/BeautySelectorAddon.ts
index 0e297c2..f9d2ef6 100644
--- a/ModLoader/mod/BeautySelectorAddon/src/BeautySelectorAddon.ts
+++ b/ModLoader/mod/BeautySelectorAddon/src/BeautySelectorAddon.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import JSZip from "jszip";
 import type {LifeTimeCircleHook, LogWrapper} from "../../../dist-BeforeSC2/ModLoadController";
 import type {AddonPluginHookPointEx} from "../../../dist-BeforeSC2/AddonPlugin";
Submodule mod/DoLTimeWrapperAddon contains modified content
diff --git a/ModLoader/mod/DoLTimeWrapperAddon/src/OldTimeFunctionHook.ts b/ModLoader/mod/DoLTimeWrapperAddon/src/OldTimeFunctionHook.ts
index 609c08a..dc50d2e 100644
--- a/ModLoader/mod/DoLTimeWrapperAddon/src/OldTimeFunctionHook.ts
+++ b/ModLoader/mod/DoLTimeWrapperAddon/src/OldTimeFunctionHook.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import {
     HookManagerCore,
     InfinityLoopChecker,
diff --git a/ModLoader/mod/DoLTimeWrapperAddon/src/TimeProxyManager.ts b/ModLoader/mod/DoLTimeWrapperAddon/src/TimeProxyManager.ts
index 00d53ce..ad46e10 100644
--- a/ModLoader/mod/DoLTimeWrapperAddon/src/TimeProxyManager.ts
+++ b/ModLoader/mod/DoLTimeWrapperAddon/src/TimeProxyManager.ts
@@ -1,3 +1,4 @@
+// @ts-nocheck
 import type {SC2DataManager} from "../../../dist-BeforeSC2/SC2DataManager";
 import type {ModUtils} from "../../../dist-BeforeSC2/Utils";
 import type {LogWrapper} from "../../../dist-BeforeSC2/ModLoadController";
